test/JDBC/expected/restricted_procedures-vu-verify.out

```diff
@@ -0,0 +1,99 @@
+-- tsql
+-- reset the login password
+ALTER LOGIN babel_4390_user_l1 WITH PASSWORD = 'abc';
+GO
```

Comment:
Is there any testcase where we are checking the superuser behaviour that it should be able to drop it?

Reply:
Didn't add the test for this case, as we'll need to recreate that dropped procedure else version upgrade will break. Tested this case locally:
```
jdbc_testdb=# show is_superuser;
 is_superuser 
--------------
 on
(1 row)

jdbc_testdb=# drop procedure master_dbo.xp_qv;
DROP PROCEDURE
```